RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
```

Comment:
Sorry, I think that our instructions are wrong (we're doing some repository consolidation at the moment and the documentation here has not caught up apparently). This file should be in the `hypothesis-python` directory, not the root of the repo.